test/integration/targets/include_vars-ad-hoc/runme.sh

```diff
@@ -4,3 +4,21 @@ set -eux
 
 ansible testhost -i ../../inventory -m include_vars -a 'dir/inc.yml' "$@"
 ansible testhost -i ../../inventory -m include_vars -a 'dir=dir' "$@"
+
+ansible-playbook playbook.yml>output.txt
```

Comment:
This should go in the normal include_vars test suite since it's not testing something adhoc specific (and is not using adhoc). Rather than grepping for defined variables, you could do this in the test play with an assert task. Other than that, LGTM.